README.md

```diff
@@ -145,7 +146,22 @@ $(eval $(call add-path,/home/user/anaconda3/envs/arkouda))
 
 The `chpl` compiler will be executed with `-I`, `-L` and an `-rpath` to each path.
 
+The minimum cmake version is 3.11.0, which is not supported in older RHEL versions such as CentOS 7; in these cases, cmake must be downloaded, installed, and linked as follows:
+
 ```
+export CM_VERSION=3.11.0
```

Comment:
While we are adding updates to verbiage that 3.11.0 was tested, it may be beneficial to have a comment here stating to use the version that you installed.